gymnasium/vector/async_vector_env.py

```diff
@@ -197,6 +197,16 @@ def __init__(
         self._state = AsyncState.DEFAULT
         self._check_spaces()
 
+    @property
+    def np_random_seed(self) -> tuple[int, ...]:
+        """Returns the seeds of the wrapped envs."""
+        return self.get_attr("np_random_seed")
+
+    @property
+    def np_random(self) -> tuple[np.random.Generator, ...]:
```

Comment:
Not sure if this is a good idea to expose, since it will (I think) be a copy of the generator, not the same object. For example, you might be tempted to do `envs.np_random.do_something()` and expect that it will affect the original vector env - but it won't.

Reply:
I agree it's not good to expose it, especially the setter - I argued for it in a previous thread.

Concerning other fixes to the vector envy - I'd prefer to do them in a separate PR.

Generally, it seems like the asynchronous discussion is going into a bit of a circle. If you want, we could have a short call to finalize this and discuss possible follow-ups in the vector envs.

I'd also be happy to discuss further integration of Gymnasium and other farama things with tianshou, if you want